protocol_driver_grpc_async_callback.cc

```diff
@@ -316,4 +323,167 @@ void ProtocolDriverGrpcAsyncCallback::ShutdownServer() {
   }
 }
 
+using grpc::Status;
```

Comment:
Removed; it's only used once and there is a risk of confusion with absl::Status.